.github/workflows/docs.yml

```
@@ -0,0 +1,77 @@
+          }
+
+      - name: 📤 upload pages artifact
+        id: artifacts
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: ./site
+  
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.artifacts.outputs.page_url }}
```

actionlint (1.7.7)

property "artifacts" is not defined in object type {deployment: {conclusion: string; outcome: string; outputs: {page_url: string}}}

(expression)